include/ttmlir/Target/Common/types.fbs

```diff
@@ -195,12 +195,24 @@ table ChipPhysicalCores {
   eth_inactive: [Dim2d];
 }
 
+enum CPURole: uint8
+{
+  Host = 0,
+  Device = 1,
+}
+
+table CPUDesc {
+  role: CPURole;
+  target_triple: string;
+}
+
 table SystemDesc {
   chip_descs: [ChipDesc];
   chip_desc_indices: [uint32];
   chip_capabilities: [ChipCapability];
   chip_coords: [ChipCoord];
   chip_channels: [ChipChannel];
+  cpu_descs: [CPUDesc];
```

Comment:
Aesthetically, I would sort of prefer to move this to top of the table to mirror what I did for the MLIR tablegen stuff.  I'm curious if 1). that would break backwards compatibility, and 2). if so if we care?
(For protobuf, appending items to the end [w.r.t. to the index number, not actual table ordering, of course] is guaranteed to be backwards-compatible, but obviously reordering existing elements would not.  Not sure if flatbuffer has similar concept and if so whether we care.

Reply:
alternatively, I think moving the cpu_descs to the front of the TableGen was actually not needed to resolve my issue, so perhaps I should just revert the order there--don't have a strong opinion on which of these is better, but would pref consistent ordering

Reply:
No strong opinion about order, but I don't think maintaining backwards compatability with older system desc flatbuffers is important right now (we're still early in project, and generating them fresh before running on silicon anyways).

Reply:
Agree with Kyle.

It definitely breaks backwards compat, but that's not important right now so we should definitely do whatever is clearer.  I think since we put the field first in tablegen, we should put it first here.